### PR TITLE
fix(scripts/RuinsOfAhnQiraj): Apply movement on Ayamiss' death

### LINKED ISSUE #12778: [AQ20] Ayamiss the Hunter does nto fall if killed on air at Phase 2
Originally reported at https://github.com/chromiecraft/chromiecraft/issues/3933

### Current Behaviour

NPC: https://wowgaming.altervista.org/aowow/?npc=15369
Ayamiss the Hunter's coprse doesn't drop to the ground if killed on air at Phase 2. Corpse stays on air and it's impossible to loot it.
image

### Expected Blizzlike Behaviour

Ayamiss the Hunter's coprse should drop to the groud if killed on air.

### Source

_No response_

### Steps to reproduce the problem

Go to AQ20 at PTR.
Ayamiss the Hunter.
Kill her on air at Phase 2.
Observe that corpse is still on air.

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/e6dd5d3e22d92dddc841079f9108275fd0359be1

### Operating system

Win 10

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by Si1ker**
## Issues Addressed:

- Closes #12778
- Closes https://github.com/chromiecraft/chromiecraft/issues/3933

## Tests Performed:

image

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Si1ker** [comment]
core-modules-build / ubuntu-20.04-clang-with-modules (pull_request) Failing after 2m

🤔?

**Comment by heyitsbench** [comment]
> 🤔?

https://github.com/azerothcore/mod-anticheat/commit/9e9105080bf3f68c606065a72ad3cf57c20c6098

**Comment by Si1ker** [comment]
> > 🤔?
> 
> [azerothcore/mod-anticheat@9e91050](https://github.com/azerothcore/mod-anticheat/commit/9e9105080bf3f68c606065a72ad3cf57c20c6098) Unrelated to the PR.

ok, ty

File: src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp [hunk 1]
```diff
@@ -177,6 +177,12 @@ struct boss_ayamiss : public BossAI
         }
     }
 
+    void JustDied(Unit* killer) override
+    {
+        me->GetMotionMaster()->MoveFall();
```
Comment (Nyeriah) [inline src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp:182]: please add a check if they are in the air phase

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**